src/v2/USTB.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.26;
+
+import {SuperstateToken} from "src/v2/SuperstateToken.sol";
+import {AllowList} from "src/AllowList.sol";
+
+/**
+ * @title USTB
+ * @notice A Pausable ERC7246 token contract that interacts with the AllowList contract to check if transfers are allowed
+ * @author Compound
```

Comment:
```suggestion
 * @author Superstate
```

Reply:
Addressed in
- https://github.com/superstateinc/ustb/pull/56/commits/4e52d672532d51249bce9677ae1801a91404eda8
- https://github.com/superstateinc/ustb/pull/56/commits/863dda0005df872dec9136d026df07f918dbb2e5